ush/config_defaults.sh

```diff
@@ -715,6 +715,7 @@ i_use_2mT4B=0
 i_T_Q_adjust=1
 l_rtma3d=.false.
 i_precip_vertical_check=0
+l_uncertainty=.false.
```

Comment:
Thanks for the changes! Since this only applies to hydrometeors, I would think it may be better to let the variable name reflect this.  How about "l_cloud_uncertainty", or "l_cld_uncertainty"?

Reply:
good idea!  I'll make that change this evening or tomorrow morning.